### DISCUSSION TITLE
How to obtain a billing after purchasing a licence ?

### DISCUSSION THREAD

**Discussion by Zemiti** [aclap-dev/video-downloadhelper#1344] [License & Purchase requests]
### What's your issue?

Dear all,
After purchasing a license through my company, they are asking me to provide a bill regarding my purchase. Is it possible please to have one. Thanks ofr your help.
Best

### The 10 first digits (only 10) of your license number or email address

99b4cc909f

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi, I sent you an email.
